swift/ql/test/library-tests/parent/child.expected

```diff
@@ -0,0 +1,1319 @@
+| declarations.swift:1:8:1:8 | init | 0 | getImmediateSelfParam() | declarations.swift:1:8:1:8 | self |
+| declarations.swift:1:8:1:8 | init | 0 | getImmediateSelfParam() | declarations.swift:1:8:1:8 | self |
```

Comment:
These are actually two different `init` functions. Once this gets merged, I have another change ready to use functions default naming for constructors and distinguish the two (namely into `init()` and `init(x:)`)